cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -1397,6 +1403,37 @@ public void bindDropletToApp(UUID dropletGuid, UUID applicationGuid) {
                 .block();
     }
 
+    @Override
+    public DropletInfo getCurrentDropletForApplication(UUID applicationGuid) {
+        GetApplicationCurrentDropletResponse getApplicationCurrentDropletResponse = delegate.applicationsV3()
+                                                                                            .getCurrentDroplet(GetApplicationCurrentDropletRequest.builder()
+                                                                                                                                                  .applicationId(applicationGuid.toString())
+                                                                                                                                                  .build())
+                                                                                            .block();
+        String packageUrl = getApplicationCurrentDropletResponse.getLinks()
+                                                                .get("package")
```

Comment:
Is there an enum or a dedicated constant for this `"package"` string?

Reply:
Moved in Contants